.github/workflows/labeler.yml

```diff
@@ -0,0 +1,62 @@
+name: "Issue Labeler"
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Add agent-java label
+      uses: actions/github-script@v7
+      with:
+        script: |
+          github.rest.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ["agent-java"]
+          })
+    - id: is_elastic_member
+      uses: elastic/apm-pipeline-library/.github/actions/is-member-elastic-org@current
+      with:
+        username: ${{ github.actor }}
+        token: ${{ secrets.APM_TECH_USER_TOKEN }}
```

Comment:
I've just applied the required changes in the IasC and this secret is now available.